cypress/integration/T3-Task.js

```diff
@@ -0,0 +1,29 @@
+import LoginPage from  '../objects/loginPage';
+import Task from '../objects/task';
+var login = new LoginPage();
+var task = new Task();
+
+describe('Sourcebooks creates task', function() {
+      it('T-3: Admin creates new task', function () {
+        task.visitTasks();
+        login.userDropDown().click({force:true});
+        login.getUser("Dainius Gaidamavičius").click();
+        login.getRoleDropDown().click({force:true});
+        login.getRoleOption("Admin").click();
+        login.getSubmitButton().click();
+        task.getCreateTaskButton().click();
+        let taskName = task.getRandomTaskName();
```

Comment:
Randomization of the task name could be done straight in the test. It's not really related to the POM, so moving it away is not required.